api/iot/iot-api.mdx

```diff
@@ -0,0 +1,13 @@
+---
+id: iot-api
+sidebar_label: Overview
+title: IoT API
```

Comment:
switch to IoT Public API to keep consistent with the headers title?
<img width="785" alt="Screen Shot 2021-06-09 at 14 14 56" src="https://user-images.githubusercontent.com/52679694/121430605-29fba100-c92d-11eb-8d01-58872a1b97b7.png">